### PR TITLE
Fix typo in thrift connection housekeeping metric name

### PR DESCRIPTION
**Pull request by ConorMcGee**
## 💸 TL;DR
Metric name typo.

## ✅ Checks

- [x] CI tests (if present) are passing
- [x] Adheres to code style for repo
- [x] Contributor License Agreement (CLA) completed if not a Reddit employee

### CODE REVIEW

File: thriftbp/prometheus.go
Comment (mterwill) [inline thriftbp/prometheus.go:126]: Updating the name of a metric is a backwards-incompatible change. Is it worth doing this in stages to give people a chance to update dashboards, etc.?

Comment (fishy) [reply thriftbp/prometheus.go:126]: this is a fairly new metric so I think it should be fine and not worth it going through the steps, unless someone feels strongly.

**Review by mterwill — APPROVED** [review]

**Review by kylelemons — APPROVED** [review]

**Comment by ConorMcGee** [comment]
I don't have write access. Happy to merge myself if someone grants it, otherwise will need someone else to do that. Thanks!

**Pull request merged**
